client/my-sites/comments/comment/style.scss

```diff
@@ -531,6 +532,14 @@
 	}
 }
 
+// Reply Visible
+
+.card.comment.is-reply-visible {
+	.comment__reply {
```

Comment:
Potentially we can move the modifier to the CommentReply component, but I can see wanting to keep this with the other modifiers at the card level.